docs/shared-content/concepts/releases.include.md

```diff
@@ -1,3 +1,3 @@
-A release is a snapshot of the deployment process and the associated assets (scripts, variables, and any references to package versions) as they existed when the release was created. The release is given a version number, and you can deploy that release as many times as you need to, even if parts of the deployment process have changed since the release was created (those changes will be included in future releases but not in this version).
+A release is a snapshot of the deployment process and the associated assets that existed when the release was created. These assets include scripts, any references to package versions, and variables (excluding [tenant variables](/docs/tenants/tenant-variables/index.md)). A version number is assigned to a release. Releases can be deployed as often as necessary, even if the deployment process has since been changed for newer releases.
```

Comment:
```suggestion
A release is a snapshot of the deployment process and the associated assets that existed when the release was created. These assets include scripts, any references to package versions, and variables (excluding [tenant variables](/docs/tenants/tenant-variables.md)). A version number is assigned to a release. Releases can be deployed as often as necessary, even if the deployment process has since been changed for newer releases.
```